src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2000,7 +2000,9 @@ namespace WaterToAirHeatPumpSimple {
 
                                 // adjust for system air flow -- capacity is based on heating design day calcs
                                 // adjust by ratio of system to heating air flow rate and temperature delta across the coil at these different airflow
-                                RatedCapCoolTotalDes *= (RatedAirVolFlowRateDes / HeatingAirVolFlowRateDes) * HeatdTratio;
+                                if (HeatingAirVolFlowRateDes > 0) {
+                                    RatedCapCoolTotalDes *= (RatedAirVolFlowRateDes / HeatingAirVolFlowRateDes) * HeatdTratio;
+                                }
```

Comment:
I ran the defect file and HeatingAirVolFlowRateDes = 0. This change protects against zone and system loads that are zero.